### ISSUE TITLE
isLocationAuthorized return error on iOS13

### ISSUE THREAD

**Issue by angelofan** [dpa99c/cordova-diagnostic-plugin#406]
# Bug report

**CHECKLIST**
- [x] I have read the [issue reporting guidelines](https://github.com/dpa99c/cordova-diagnostic-plugin#reporting-issues)

- [x] I confirm this is a suspected bug or issue that will affect other users

- [x] I have reproduced the issue using the [example project](https://github.com/dpa99c/cordova-diagnostic-plugin-example) or provided the necessary information to reproduce the issue.

- [x] I have read [the documentation](https://github.com/dpa99c/cordova-diagnostic-plugin) thoroughly and it does not help solve my issue.

- [x] I have checked that no similar issues (open or closed) already exist.

**Current behavior:**

Gettiing the following error after calling 'isLocationAuthorized' method on iOS 13 device:
```
[CLLocationManager authorizationStatus]:
unrecognized selector sent to instance 0x283b0a190
```

**Expected behavior:**

Return successful callback

**Steps to reproduce:**

Run [example project](https://github.com/dpa99c/cordova-diagnostic-plugin-example) on iOS 13

**Screenshots**

![image](https://user-images.githubusercontent.com/20255750/88154305-39043280-cc39-11ea-8fd4-ca6d582b4614.png)

**Environment information**

- Cordova CLI version: 9.0.0
- Cordova platform version: cordova-ios@5.1.1
- Plugins & versions installed in project (including this plugin)
```
cordova-open-native-settings 1.5.2 "Native settings"
cordova-plugin-advanced-http 2.0.6 "Advanced HTTP plugin"
cordova-plugin-androidx 2.0.0 "cordova-plugin-androidx"
cordova-plugin-androidx-adapter 1.1.1 "cordova-plugin-androidx-adapter"
cordova-plugin-app-version 0.1.9 "AppVersion"
cordova-plugin-badge 0.8.8 "Badge"
cordova-plugin-ble-central 1.2.5 "BLE"
cordova-plugin-chrome-apps-common 1.0.7 "Chrome Apps Common Utils"
cordova-plugin-chrome-apps-iossocketscommon 1.0.2 "Chrome Apps iOS Sockets Common"
cordova-plugin-chrome-apps-sockets-tcp 1.3.7 "Chrome Apps Sockets TCP API"
cordova-plugin-device 2.0.3 "Device"
cordova-plugin-esptouch 1.0.8 "Cordova esptouch"
cordova-plugin-file 6.0.1 "File"
cordova-plugin-inappbrowser 3.2.0 "InAppBrowser"
cordova-plugin-ionic-keyboard 2.1.3 "cordova-plugin-ionic-keyboard"
cordova-plugin-ionic-webview 2.4.0 "cordova-plugin-ionic-webview"
cordova-plugin-ios-localized-strings 1.0.6 "Cordova iOS Localized Strings"
cordova-plugin-localization-strings 3.1.0 "Localization"
cordova-plugin-nativeaudio 3.0.9 "Cordova Native Audio"
cordova-plugin-nativeclicksound 0.0.4 "Native Click Sound"
cordova-plugin-nativestorage 2.3.2 "NativeStorage"
cordova-plugin-network-information 2.0.1 "Network Information"
cordova-plugin-networkinterface 2.0.0 "NetworkInterface"
cordova-plugin-ping 0.5.0 "Ping"
cordova-plugin-screen-orientation 3.0.1 "Screen Orientation"
cordova-plugin-splashscreen 5.0.2 "Splashscreen"
cordova-plugin-statusbar 2.4.3 "StatusBar"
cordova-plugin-vibration 3.1.0 "Vibration"
cordova-plugin-whitelist 1.3.3 "Whitelist"
cordova-support-google-services 1.3.2 "cordova-support-google-services"
cordova.plugin.HttpAes256Key16 1.0.2 "HttpAes256Key16"
cordova.plugins.diagnostic 6.0.0 "Diagnostic"
es6-promise-plugin 4.1.0 "Promise"
phonegap-plugin-multidex 1.0.0 "Multidex"
phonegap-plugin-push 2.3.0 "PushPlugin"
wifiwizard2 3.1.1 "WifiWizard2"
```
- Dev machine OS and version
```
ProductName:	Mac OS X
ProductVersion:	10.15.5
BuildVersion:	19F101
```
        
_Runtime issue_
- Device details: iPhone 7
- OS details: iOS 13.5.1

**Comment by angelofan** [comment]
It runs fine on the iPhone 8 simulator of iOS 13.6, and it seems like a bug in iOS 13.5.1.

**Comment by angelofan** [comment]
I upgraded the system version of the real device with this problem to iOS 13.6. The problem still exists. So far I am not sure what caused this error.

I will use more colleagues’ iPhones to test this problem.

**Comment by angelofan** [comment]
I ran the sample project on iOS 13.4.1 on another colleague’s iPhone 11, and still had the same problem, unable to detect location authorization status.

**Comment by angelofan** [comment]
After I removed the plug-in and reinstalled version 5.0.2, they all worked normally.

It always works fine in the simulator, and cannot call `isLocationAuthorized` in iOS 13 on a real device.

**Comment by dpa99c** [comment]
Thanks for reporting - I must have missed this bug in regression testing because I tested in the Simulator rather than on a device.
I'll investigate and fix in a patch version.

**Comment by dpa99c** [comment]
The above fix has been published to npm in patch version `6.0.1` - this should resolve your issue but let me know if you still encounter problems.
